addons/acr/units/CfgUnits.hpp

```diff
@@ -0,0 +1,207 @@
+//Units
+class GVAR(Rifleman): GVAR(Soldier_Base) {
+    scope = 2;
+    faction = QUOTE(ADDON);
+    side = 0;
```

Comment:
Faction and side are specified in the base class in CfgVehicles, don't need to repeat each time